src/project/mod.rs

```diff
@@ -416,6 +417,11 @@ impl Project {
         self.manifest.has_pypi_dependencies()
     }
 
+    /// Returns the custom location of pypi-name-mapping
+    pub fn custom_pypi_mapping(&self) -> MappingSource {
```

Comment:
Same here, also just call this `pypi_name_mapping_source` or something like that?